src/DucksNet.WebUI/Pages/Models/CreateEmployeeModel.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace DucksNet.WebUI.Pages.Models;
+
+public class CreateEmployeeModel
+{
+    [Required]
+    [StringLength(36, MinimumLength = 36, ErrorMessage = "Guid must be 36 characters long.")]
+    public string? IdOffice { get; set; }
```

Comment:
If I put a bad Id in here the request fails with 400 ( wich is expected ) but even if I put a good ID it will create an employee with Guid.Empty. I suspect you're doing a `new Guid()` instead of `Guid.NewGuid()` somewhere.

Reply:
In swagger, the information is updated, but in model, I need to add a new member Employee which remembers the Id. It is a TODO.